app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -462,10 +462,10 @@ public Fragment getItem(int pos) {
                     return FilesystemViewerFragment.newInstance(getFilesystemFragmentOptions(null));
                 }
                 case R.id.nav_quicknote: {
-                    return DocumentEditFragment.newInstance(_appSettings.getQuickNoteFile(), -1);
+                    return DocumentEditFragment.newInstance(_appSettings.getQuickNoteFile(), -1, MainActivity.this);
                 }
                 case R.id.nav_todo: {
-                    return DocumentEditFragment.newInstance(_appSettings.getTodoFile(), -1);
+                    return DocumentEditFragment.newInstance(_appSettings.getTodoFile(), -1, MainActivity.this);
```

Comment:
Can't we just use getActivity() for the parent? We already use `getActivity() instanceof MainActivity` etc

Reply:
> Can't we just use getActivity() for the parent? We already use `getActivity() instanceof MainActivity` etc

Thanks. I didn't know android has this method for Fragment before.
You are right, no need for a new field.